lib/libtlspool.c

```diff
@@ -409,31 +263,11 @@ static int np_send_command(struct tlspool_command *cmd) {
  * TLS Pool anew, using exponential back-off.
  */
 static void *master_thread (void *path) {
-#ifndef WINDOWS_PORT
-	useconds_t usec;
-	struct sockaddr_un sun;
-	//NOT-USED// char anc [CMSG_SPACE(sizeof (int))];
-	struct iovec iov;
-	struct cmsghdr *cmsg;
-	struct msghdr mh = { 0 };
-#else
```

Comment:
kan deze extra rommel echt weg?  doordat je het in een andere routine stopt neem ik aan.  netjes :)